woocommerce/woocommerce-ios
# [Store onboarding] Option to hide the list

Part of: #9561


## Description

We want to allow the user to hide/dismiss the Store onboarding task list.

This PR,

- Adds an option to store onboarding list to hide it. 
- In a later PR we will add a way to enable this from settings.

## Testing instructions

- Enable feature flag by returning `true` [here](https://github.com/woocommerce/woocommerce-ios/compare/feat/9561-option-to-hide?expand=1#diff-26c9afc49bb1f1a168517d43d5e36a64ff45adb046469369ef3e4d6800ccc19bR97)
- Login into a store which has pending store onboarding tasks
- From the dashboard store onboarding list tap on `...` -> and select `Hide store setup list`
- From the alert tap `Remove` and verify that the store onboarding task list is removed.

## Screenshots

![Simulator Screen Recording - iPhone 14 - 2023-04-28 at 07 17 11](https://user-images.githubusercontent.com/524475/235035573-a37640f0-f854-4c20-a6b0-133a18cc2670.gif)


---
